packages/utils-general/src/OriginFeature.ts

```diff
@@ -8,7 +8,8 @@ export enum OriginFeature {
     Devices = 'devices',
     Certificates = 'certificates',
     CertificationRequests = 'certificationRequests',
-    IRec = 'irec'
+    IRec = 'irec',
+    IRecConnect = 'irecConnect'
```

Comment:
Why do we need to have a separate OriginFeature `IRecConnect`? Isn't IRecConnect already a part of the larger IRec feature?

Reply:
To better formulate my question, is there some case where we will have `IRec` turned **OFF**, and `IRecConnect` turned **ON**? I assume not.

Reply:
Now we don't have the backend part for IRec Connect, because it is not clear how exactly the connection with existing I-Rec account will happen.We have only the "Register new IRec Account" part. That's why we should keep I-Rec connect aside for now.